area/area.go

```diff
@@ -125,3 +129,41 @@ func (m *GormAreaRepository) ListChildren(ctx context.Context, parentArea *Area)
 	}
 	return objs, nil
 }
+
+// Query exposes an open ended Query model for Area
+func (m *GormAreaRepository) Query(funcs ...func(*gorm.DB) *gorm.DB) ([]*Area, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "area", "query"}, time.Now())
+	var objs []*Area
+
+	err := m.db.Scopes(funcs...).Table(m.TableName()).Find(&objs).Error
+	if err != nil && err != gorm.ErrRecordNotFound {
+		return nil, errs.WithStack(err)
+	}
+
+	log.Debug(nil, map[string]interface{}{
+		"areaList": objs,
```

Comment:
We are progressively changing the format of the attributes to use lowercase and snake case. So, better use `"area_list": objs`.

We chose to use this approach cause goa default logs use this format. So we keep some consistency on it.